rcsb/workflow/tests/testBcifWorkflow.py

```diff
@@ -0,0 +1,698 @@
+##
+# File:    testBcifWorkflow.py
+# Author:  James Smith
+# Date:    6-Mar-2025
+##
+
+"""
+Test bcif workflow with python unit test framework.
+"""
+
+__docformat__ = "google en"
+__author__ = "James Smith"
+__email__ = "james.smith@rcsb.org"
+__license__ = "Apache 2.0"
+
+import os
+import shutil
+import glob
+import gzip
+import tempfile
+import unittest
+from typing import List
+import logging
+import time
+from rcsb.workflow.bcif.task_functions import (
+    convert,
+    deconvert,
+    getDictionaryApi,
+)
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s [%(levelname)s]-%(module)s.%(funcName)s: %(message)s",
+)
+
```

Comment:
Here, for this `testBcifWorkflow` file, I think you want to _keep_ this, but should adjust it to as follows:

```suggestion
logging.basicConfig(
    level=logging.INFO,
    format="%(asctime)s [%(levelname)s]-%(module)s.%(funcName)s: %(message)s",
)
logger = logging.getLogger()
logger.setLevel(logging.INFO)
```

Then for all calls to `logging.info` or `.warning` etc. below, you'd change them to `logger.info` etc.